src/steps/owner.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  createIntegrationEntity,
+  IntegrationStep,
+  IntegrationStepExecutionContext,
+} from '@jupiterone/integration-sdk-core';
+import { createAPIClient } from '../client';
+import { IntegrationConfig } from '../config';
+
+export async function fetchOwners({
+  jobState,
+  instance,
+}: IntegrationStepExecutionContext<IntegrationConfig>) {
+  const apiClient = createAPIClient(instance.config);
+  await apiClient.iterateOwners(async (owner) => {
+    await jobState.addEntity(
+      createIntegrationEntity({
+        entityData: {
+          source: {
+            id: owner.id,
+          },
+          assign: {
+            _key: `hubspot-owner-${owner.id}`,
+            _type: 'acme_user',
+            _class: 'User',
+            // This is a custom property that is not a part of the data model class
+            // hierarchy. See: https://github.com/JupiterOne/data-model/blob/master/src/schemas/User.json
+            firstName: owner.firstName,
+            // lastName: owner.lastName,
+            name: `${owner.firstName} ${owner.lastName}`,
+            email: owner.email,
+            username: owner.email,
+          },
+        },
+      }),
```

Comment:
Let's use this newer pattern of creating folders for separate steps.
So in this example, we'll have `src/steps/owners` folder.

Inside that folder, we'll have a couple of files:
1) `index.ts` - is what you've got inside `owner.ts` currently
2) `index.test.ts` - is what you've got inside `index.test.ts` currently
3) `converters.ts` - we can have a converter for the Owner here (e.g. looks like this https://github.com/JupiterOne/graph-azure/blob/master/src/steps/resource-manager/batch/converters.ts#L16, you'd just port the code you've got inside `owner.ts`)
4) `converters.test.ts` - tests for converters, we can go over this in the next PR/over Slack
5) `constants.ts` - you'll move the definition for `_type` and `_class` here and then just reference them via variable instead of using it directly (e.g. owner.ts L46-47 currently).

Here's an example of how it should look like: https://github.com/JupiterOne/graph-azure/tree/master/src/steps/resource-manager/batch.